requirements.txt

```
@@ -1,4 +1,4 @@
-cx_Oracle~=7.0
-Flask~=1.0
-mysqlclient~=2.0
-psycopg2~=2.0
+oracledb~=1.3
+Flask>2.0
+mysqlclient>2.0
+psycopg2>3.0
```

See https://www.psycopg.org/psycopg3/docs/basic/install.html#binary-installation

```suggestion
psycopg[binary]>3.0
```